### ISSUE TITLE
Gallery examples use old import convention.

### ISSUE THREAD

**Issue by mkcor** [scikit-image/scikit-image#7454]
### Description:

We should `import skimage as ski` in all gallery examples!

This issue may (should) be addressed in small batches of examples.

**Comment by jgyasu** [comment]
Hi @mkcor, I am first-time contributor to scikit-image, can you assign this to me?

**Comment by mkcor** [comment]
Hi @jgyasu, feel free to start with an example of your choice, mentioning this issue in the PR description. Let us know if you run into any trouble along the way. It wouldn't make much sense to assign this issue to a single contributor, since

> This issue may (should) be addressed in small batches of examples.

**Comment by jgyasu** [comment]
@mkcor thank you! Can you please show me an example or point me to one where the old import convention is used? I will get the idea then

**Comment by mkcor** [comment]
Yes... Well, have you browsed the [gallery of examples](https://scikit-image.org/docs/stable/auto_examples/)? When you do so, you will notice that most examples use the old import convention. Visit, say, the [first example](https://scikit-image.org/docs/stable/auto_examples/data/plot_3d.html#sphx-glr-auto-examples-data-plot-3d-py): You can see that the code starts with
```py
from skimage import data
```

Have you forked the repo and cloned your fork locally? See: https://scikit-image.org/docs/stable/development/contribute.html

Then, are you familiar with any recursive grep tool, so you can find a given character string in the code base? Note that you can also search the code base from the GitHub interface: https://github.com/scikit-image/scikit-image

**Comment by jgyasu** [comment]
I went through the example, and from what I understand, currently it is something like this,

```
from skimage import data
import plotly
import plotly.express as px
import numpy as np

img = data.cells3d()[20:]
...
```

And according to the new convention it should be,

```
import skimage as ski
import plotly
import plotly.express as px
import numpy as np

img = ski.data.cells3d()[20:]
...
```
right?

And yes, I have forked and cloned the repo locally and I am familiar with recursive grep tools. Thanks again.

**Comment by AlexLouk** [comment]
Hello has this Issue been resolved ?

**Comment by mkcor** [comment]
Hi @AlexLouk,

Not entirely! Please browse the [gallery](https://scikit-image.org/docs/dev/auto_examples) and you will see that many examples haven't been updated yet. For example, this one:  https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_metrics.html Are you willing to update it? 😃

**Comment by AlexLouk** [comment]
Sure, I will look into it :)

**Comment by AlexLouk** [comment]
This Issue is a pretty good starter and I have changed a couple of the imports now. I would maybe leave it open to encourage more people to start contributing if u agree. If not I can change the other imports too so we can close the issue @mkcor.

**Comment by mkcor** [comment]
@AlexLouk all good, I agree (for now)! Thanks for submitting #7764.
